tests/intermittents_commenter/expected_comment_with_5_failures.text

```diff
@@ -0,0 +1,30 @@
+## Summary
+5 failures in 10 pushes (0.5 failures/push) were associated with this bug in the last 7 days.
+
+This is the #1 most frequent failure this week.
+
+## Repository breakdown:
+* mozilla-central: 5
+
+## Platform and build breakdown:
+* b2g-emu-jb: 2
+  * debug: 1
+  * opt: 1
+* osx-10-6: 1
+  * debug: 1
+* osx-10-7: 1
+  * debug: 1
+* windows7-32: 1
+  * debug: 1
+
+## Table
+| |B2G Emulator Image Build|Mochitest Browser Chrome|
+|---|:-:|:-:|
+|b2g-emu-jb/debug|1| |
```

Comment:
Should the number not be in the last cell (for debug)?

Reply:
No, I don't think so. It corresponds to the first job in the file https://github.com/mozilla/treeherder/blob/master/tests/sample_data/job_data.txt . 
What makes you think that?

Reply:
I am looking into this today- the data in the file is years old, but should still be somewhat relevant.  First look is we will want rows to be `platform:config`, and columns to be `test_variants` - but there is no clean way to define test variants; we can make progress now and refine in round 2.

Reply:
as commented in the specific code review- we need to:
1) update the sample data with better task names, get rid of really old stuff, and add stuff with variants
2) use code (provided sample) to find variants from the task name!

given those 2, I think the table fields will be more logical and concerns will go away!  Thanks for working on this- while it seems "minor", I think it is a very big change for something that hasn't changed in 6+ years.